omnibus/config/projects/push-jobs-client.rb

```diff
@@ -35,12 +35,14 @@
   install_dir "#{default_root}/#{name}"
 end
 
-override :bundler,        version: "1.7.12"
 # Uncomment to pin the chef version
 #override :chef,           version: "12.2.1"
```

Comment:
Also - chef (our software definition) pulls in ohai which also picks up master by default.  Pin it to whatever chef pinned it to when it was released.